README.md

```diff
@@ -1,76 +1,81 @@
 <h3 align="center">
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-	Catppuccin for <a href="https://github.com/vendetta-mod/Vendetta">Vendetta</a>
+	Catppuccin for <a href="https://github.com/pyoncord/Bunny">Bunny</a> & <a href="https://github.com/enmity-mod/enmity">Enmity</a>
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
 </h3>
 
 <p align="center">
-	<a href="https://github.com/riivx/vendetta/stargazers"><img src="https://img.shields.io/github/stars/riivx/vendetta?colorA=363a4f&colorB=b7bdf8&style=for-the-badge"></a>
-	<a href="https://github.com/riivx/vendetta/issues"><img src="https://img.shields.io/github/issues/riivx/vendetta?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
-	<a href="https://github.com/riivx/vendetta/contributors"><img src="https://img.shields.io/github/contributors/riivx/vendetta?colorA=363a4f&colorB=a6da95&style=for-the-badge"></a>
+	<a href="https://github.com/riivx/vendetta/issues?q=is%3Aissue+is%3Aopen+label%3Amobile"><img src="https://img.shields.io/github/issues/riivx/vendetta?colorA=363a4f&colorB=f5a97f&style=for-the-badge"></a>
 </p>
 
 <p align="center">
-	<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/preview.webp"/>
+  <img src="assets/preview.webp"/>
 </p>
 
 ## Previews
 
 <details>
 <summary>🌻 Latte</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/latte.webp"/>
+<img src="assets/latte.webp"/>
 </details>
 <details>
 <summary>🪴 Frappé</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/frappe.webp"/>
+<img src="assets/frappe.webp"/>
 </details>
 <details>
 <summary>🌺 Macchiato</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/macchiato.webp"/>
+<img src="assets/macchiato.webp"/>
 </details>
 <details>
-<summary>🌿 Mocha</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta//main/assets/mocha.webp"/>
+<summary>🌿 Mocha</summary> 
+<img src="assets/mocha.webp"/>
 </details>
 
 ## Usage
 
->[!WARNING]
->Vendetta was **discontinued** on February 7. 2024. Use [Bunny](https://github.com/pyoncord/Bunny) instead. It supports Vendetta themes
-
```

Comment:
I think we should keep this text but reword it to explain the point of this repository, and under `[!IMPORTANT]` instead.